README.md

```diff
@@ -25,6 +27,20 @@ runs:
 
 These setup steps are run after the repository has been checked out and after Go has been installed, but before any tests or checks are run.
 
+### Configuration
+
+`go-check` contains an optional step that checks that running `go generate` doesn't change any files.
+This is useful to make sure that the generated code stays in sync, however, it requires code generators to be deterministic.
+
+This check will be run in repositories that set `gogenerate` to `true` in `.github/workflows/go-check-config.json`:
+```json
+{
+  "gogenerate": true
+}
+```
+
+Note that depending on the code generators used, it might be necessary to [install those first](#additional-setup-steps).
```

Comment:
Maybe you can put the determinism note here instead, as a second sentence:

> The generators must also be deterministic, to prevent CI from getting different results each time.

Reply:
Thanks. Applied.